python/lvmdrp/functions/run_drp.py

```diff
@@ -334,7 +334,7 @@ def reduce_frame(filename: str, camera: str = None, mjd: int = None,
         kwargs = get_config_options('reduction_steps.determine_wavesol')
         log.info('--- Determining wavelength solution ---')
         log.info(f'custom configuration parameters for determine_wave_solution: {repr(kwargs)}')
-        determine_wavelength_solution(in_arc=marc, out_wave=wave_file, out_lsf=lsf_file,
+        determine_wavelength_solution(in_arc=xout_file, out_wave=wave_file, out_lsf=lsf_file,
```

Comment:
why is this changed from `marc` to `xout_file`?  I thought this step is only run for the master arc files?

Reply:
We determine wavelength solutions on the (master) **extracted** arc frames. If you try to determine wavelength solutions 2D images, you'll get a lot of errors in the fitting just because > half of the rows are just noise.

Reply:
ahh yes, that's right!  good catch.